src/platform/forms-system/src/js/components/FormNavButtons.jsx

```diff
@@ -30,6 +30,7 @@ const FormNavButtons = ({ goBack, goForward, submitToContinue }) => (
         buttonText="Continue"
         buttonClass="usa-button-primary"
         afterText="»"
+        ariaDescribedBy="nav-form-header"
```

Comment:
It's not obvious where this element lives. We're referencing it here in `FormNavButtons`, but it is defined in `FormNav`:

https://github.com/department-of-veterans-affairs/vets-website/blob/061702c0338433cdfad84f4f0aaa57a16b2405af/src/platform/forms-system/src/js/components/FormNav.jsx#L102

Can you add a comment explaining the relationship between these components so that someone doesn't delete this by accident thinking that it is leftover code?

---

Also, is there a reason why we're adding `ariaDescribedBy` to the "Continue" button but not the "Back" button?

Reply:
Nice catch, @bkjohnson. I've added ariaDescribedBy to the back button as well.